### PR TITLE
Fix notes button not being visible unless hovered

### LINKED ISSUE #691: [Feature] Show note icon when account has notes
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

When a budget category has a note, the note icon is always visible, like this:
image

Account notes were recently added, but the icon only appears on hover even if there is a note present, like this:
image
image

### Describe your ideal solution to this problem

Account notes should behave the same way category notes do, displaying the icon all the time if the note is non-empty.

### Teaching and learning

_No response_

### PR DESCRIPTION
**Pull request by j-f1**
Fixes #691.

### CODE REVIEW

**Comment by MatissJanis** [comment]
🐛 the notes icon is not visible in the budgets page if there is content. On the master version the notes icon is always visible if there is content.

**Comment by j-f1** [comment]
I’m not sure I understand, can you rephrase that with a bit more detail @MatissJanis?

**Comment by MatissJanis** [comment]
Sure.

On master branch: https://actualbudget.netlify.app/budget

Screenshot 2023-02-27 at 18 03 55
Screenshot 2023-02-27 at 18 04 08

--

On preview deployment:

Screenshot 2023-02-27 at 18 04 38
Screenshot 2023-02-27 at 18 04 45

**Comment by j-f1** [comment]
Got it, looks like using explicit CSS selectors (like `& button`) override the properties set directly on the element using `style={}`.

**Review by MatissJanis — APPROVED** [review]
All good now!
